webpack/assets/javascripts/react_app/components/SettingUpdateModal/components/SettingForm/SettingForm.js

```diff
@@ -28,6 +28,7 @@ const SettingForm = ({
       message: __('Setting was successfully updated.'),
       method: 'put',
       successCallback: setModalClosed,
+      errorToast: error => `${__('Error updating setting')}: ${error.message}`,
```

Comment:
Is the actual error getting eaten out in your toast notification?

Reply:
The `error.message` in the API response does not seem to be exposed in the toast notification. (see above) If you know how to fix it please share :smile:

Reply:
```      errorToast: error => `${__('Error updating setting')}: ${error.response.data.error.message}`,```
seemed to work for me :)